client/common/directives/ModelFieldPointsOfInterest/ModelFieldPointsOfInterest.js

```diff
@@ -359,7 +361,7 @@ angular.module('dashboard.directives.ModelFieldPointsOfInterest', [
 							scope.displayedSearchResults.push(result);
 						}
 					}
-					if (scope.data.placeId) { // pre-select point if exist
+					if (scope.data.placeId && !scope.initalLoad) { // pre-select point if exist
```

Comment:
How come we don't want it on the initialLoad?

Reply:
The reason why we need to skip the initial loading is because we want to display the data from "Our" database rather than the data from the "Maps". Scenarios for this is, if the address was edited by the user, then saved, when the modal was loaded again, data from "Map" will be displayed from the form, and not the data from "Our" database[Edited Data].